docs/userGuide/cliCommands.md

```diff
@@ -9,13 +10,13 @@
 
 MarkBind Command Line Interface (CLI) can be run in the following ways:
 
-* `markbind [--version | -v]`<br>
+* `markbind [--version | -V]`<br>
   Prints the version number of the MarkBind installation in use.<br>
-  {{ icon_examples }} `markbind --version`, `markbind -v`
+  {{ icon_examples }} `markbind --version`, `markbind -V`
 * `markbind [--help | -h]`<br>
   Prints a summary of MarkBind commands.<br>
   {{ icon_examples }} `markbind --help`, `markbind -h`
-* `markbind <command> [--help | -h]`
+* `markbind <command> [--help | -h]`<br>
```

Comment:
Let's consolidate `--help`  in this Overview section since it's elaborated below.
Shall we remove short flags for simplicity — less clutter for quick reference?

`markbind [command] --help`

---

Actually, how about replacing the content of this section with the raw output of `markbind --help`?
I find it much more concise. Inspired by https://github.com/tj/commander.js/#automated---help.

```
$ markbind --help
Usage: index  <command>

Options:
  -V, --version                      output the version number
  -h, --help                         output usage information

Commands:
  init|i [root]                      init a markbind website project
  serve|s [options] [root]           build then serve a website from a directory
  deploy|d [options]                 deploy the site to the repo's Github pages.
  build|b [options] [root] [output]  build a website
```

Also, let's use the same convention as Commander.js for optional arguments:
- `[<args>]` -> `[args]`
- `[<OPTIONS>]` -> `[options]`

Reply:
> Let's consolidate --help in this Overview section since it's elaborated below.

So consolidate the summaries into something like `markbind [command] [--help | -h]`?

> Shall we remove short flags for simplicity — less clutter for quick reference?

I was trying to be consistent with the documentation for `--version`. Should we remove the alias for that in the summary as well?

> Actually, how about replacing the content of this section with the raw output of `markbind --help`?

Did you mean in the detailed section below or in the command summary here? On that note, I noticed that the "usage" line has an additional space prior to `<command>`. Could I make the corrections in this PR or should I open a new issue for that? It's just the removal of one space from `index.js`

> Also, let's use the same convention as Commander.js for optional arguments:

Did you mean just for the help command, or the detailed explanations of other commands in this page? I can go about making changes if it's the latter :)

Reply:
1. > So consolidate the summaries into something like `markbind [command] [--help | -h]`?

Yes, preferably `markbind [command] --help` since `--help` is actually not optional in this case.

2. > I was trying to be consistent with the documentation for `--version`. Should we remove the alias for that in the summary as well?

Yes, I see that you were being consistent. And yes, I suggest removing the alias for that as well.

3. >> Actually, how about replacing the content of this section with the raw output of `markbind --help`?
   >
   > Did you mean in the detailed section below or in the command summary here?

*This section* is the Overview section, also "the command summary here".
Oh, if we proceed with this, then points 1 and 2 above become irrelevant.

4. > On that note, I noticed that the "usage" line has an additional space prior to `<command>`. Could I make the corrections in this PR or should I open a new issue for that? It's just the removal of one space from `index.js`

I noticed that too. You can make the corrections in this PR.
Let's also remove the period at the end of the `pages.` line.

5. >> Also, let's use the same convention as Commander.js for optional arguments:
   >
   > Did you mean just for the help command, or the detailed explanations of other commands in this page? I can go about making changes if it's the latter :)

I mean the latter.

Reply:
> This section is the Overview section, also "the command summary here".
Oh, if we proceed with this, then points 1 and 2 above become irrelevant.

Would you prefer to have the current format or the output of `markbind --help` being displayed at the command summary then? I'm thinking the latter might be more appropriate in the detailed usage section

Reply:
I've indicated my preference twice in no uncertain terms, so I don't understand the confusion.

Why do you think the output of `markbind --help` might be more appropriate in the detailed usage section?

Reply:
> I've indicated my preference twice in no uncertain terms, so I don't understand the confusion.

Apologies for the confusion, I assumed from points 1 & 2 above and your statement "if we proceed" as an indication that you had not yet decided. 

> Why do you think the output of `markbind --help` might be more appropriate in the detailed usage section?

I figured it'd keep the command summary concise and consistent, but I realise now you meant to replace the entire summary/section with the output of `--help`. I misread your comment and thought you wanted to replace the `--help` portion in that section not the entire section. This was my misunderstanding apologies again, thank you for your patience and clarification. I'll go about making the necessary changes if that's okay :)

Reply:
Ah yes. Thanks for clarifying too.

I said "if we proceed" because I am making a recommendation that you may disagree on.
This is a team/community effort; I appreciate your contribution and welcome your opinion.

Please proceed if you agree with the changes.

Reply:
Yeap okay gotcha! Thanks again for all the help, really appreciate it :)